src/aap_eda/tasks/orchestrator.py

```diff
@@ -230,31 +229,45 @@ def dispatch(
             # The are no healthy queues on which to run the request. For any
             # request that isn't a restart set its status to "workers offline."
             msg = (
-                "There are no healthy queues on which to run the request"
-                f"{request_type_text} for {process_parent_type}"
-                f" {process_parent_id}. There may be an issue with the node"
-                "; please contact the administrator."
+                f"The workers available to process request type {request_type}"
+                f" for {process_parent_type} {process_parent_id}"
+                " are failing liveness checks.  There may be an issue with the"
+                " node; please contact the administrator."
             )
             LOGGER.warning(
                 msg,
             )
 
-            if request_type not in [
-                ActivationRequest.RESTART,
-            ]:
-                status_manager = StatusManager(
-                    get_process_parent(process_parent_type, process_parent_id),
-                )
-                status_manager.set_status(
-                    ActivationStatus.WORKERS_OFFLINE,
-                    msg,
-                )
-                # There may not be a latest instance.
-                if status_manager.latest_instance:
-                    status_manager.set_latest_instance_status(
-                        ActivationStatus.WORKERS_OFFLINE,
-                        msg,
-                    )
+            status_manager = StatusManager(
+                get_process_parent(process_parent_type, process_parent_id),
+            )
+
+            status = ActivationStatus.WORKERS_OFFLINE
+            # If the request is start/auto-start we know the request isn't
+            # running.
+            # If the request is in one of the pending states that also
+            # indicates it's not running; this check is for previous
+            # failures to start that got dispatched from the monitor
+            # task.
+            if (
+                request_type
+                in [
+                    ActivationRequest.START,
+                    ActivationRequest.AUTO_START,
+                ]
+            ) or (
+                status_manager.db_instance.status
+                in [
+                    ActivationStatus.PENDING,
+                    ActivationStatus.PENDING_WORKERS_OFFLINE,
+                ]
+            ):
```

Comment:
This is kind of nitpicky and my python idioms aren't great so take my suggestion with a grain of salt, but would it make sense to wrap this conditional in a private method with a simple name? (Or is this ugly because of the need to pass in the parameters?)
```python
if __request_is_not_currently_running(request_type, status_manager.db_instance.status):
```

Reply:
We've got two independent conditions and we're manipulating an external object.  I don't think it's significantly better/worse either way; we're still going to want to explain why those conditions (as per the comment) are significant.

However, moving the whole thing (such that status_manager doesn't even appear in this location) to a separate method would be reasonable.